paymentsheet/src/main/java/com/stripe/android/customersheet/CustomerSheetLoader.kt

```diff
@@ -203,7 +202,7 @@ internal class DefaultCustomerSheetLoader(
         val supported = setOfNotNull(
             PaymentMethod.Type.Card.code,
             PaymentMethod.Type.USBankAccount.code.takeIf {
-                customerSheetACHv2.isEnabled && isFinancialConnectionsAvailable()
```

Comment:
We should double check this, but if FC isn't linked, we shouldn't get to this point with it. So we shouldn't need the takeIf at all.

Reply:
Verified that it is handled by the LPM foundations update (through `PaymentMethodMetadata`). We also have a few tests in `DefaultCustomerSheetLoader` to ensure this behaviour is maintained.